packages/runtime/datastore/src/dataStoreRuntime.ts

```diff
@@ -202,6 +202,9 @@ IFluidDataStoreChannel, IFluidDataStoreRuntime, IFluidHandleContext {
         // Must always receive the data store type inside of the attributes
         if (tree?.trees !== undefined) {
             Object.keys(tree.trees).forEach((path) => {
+                // Issue #4414
```

Comment:
Can you please add a comment why we need to skip search blobs. Also, I think a link to the issue would be better as it makes it easier to navigate.